meta/main.yml

```diff
@@ -5,14 +5,14 @@ galaxy_info:
   description: An Ansible role for installing Open Trip Planner.
   company: Azavea Inc.
   license: Apache
-  min_ansible_version: 1.5
+  min_ansible_version: 2.0
   platforms:
   - name: Ubuntu
     versions:
-    - trusty
+    - xenial
```

Comment:
Is the OS upgrade a requirement of the software upgrade? Ideally, we wouldn't drop support for an OS version we currently use, but instead try to add support in a way that is backwards compatible.